data/json/items/comestibles/carnivore.json

```diff
@@ -118,7 +118,7 @@
     "copy-from": "gastropod_purged",
     "type": "COMESTIBLE",
     "name": { "str_sp": "cooked gastropod" },
-    "description": "Freshly cooked gastropod.  A bit low on calories but high in protein.",
+    "description": "Escargots de Cataclysm.  A bit low on calories, but high in protein.",
```

Comment:
Is it really better to replace a simple description with a weird pseudo french name?

Reply:
My logic at the time is that the current recipe for making this item specifically requires cooking several normal-size snails, essentially making proper escargots instead of a "chunk of gastropod meat". Looking at it now I don't love the change anymore, and a more generic description helps future-proof it.